src/jupyter_contrib_nbextensions/nbextensions/toc2/toc2.yaml

```diff
@@ -39,7 +39,7 @@ Parameters:
 - name: toc2.moveMenuLeft
   description: Move notebook's title and menu on the left instead of being centered -- This provides a better look when the toc/sidebar is present
   input_type: checkbox
-  default: true    
+  default: true      
```

Comment:
can we lose the trailing whitespace here please?
In fact, anywhere there is any, I guess :wink: